ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -322,6 +465,29 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
+- as: azure-ipi-des-day2-arm-f28
```

Comment:
I do not think we need to cover the day2 profile, we already have ` azure-ipi-des-mini-perm` to cover it. cc @jinyunma to double confirm.

Reply:
agree, `azure-ipi-des-mini-perm-fips` covers more.
`azure-ipi-des-day2-arm-f28` only enables encryption at host in day2 by using system-managed key, 
`azure-ipi-des-mini-perm-fips` enables encryption disk by using customer-managed keys, and also set des into default storageclass.

Reply:
Thanks. Removed.

Reply:
Hi @jianlinliu I'd like to confirm further. Do all the day2 profiles not need upgrade jobs? Or it's specific to this profile only.

Reply:
In any situation, I can NOT say we should always filter out some keyword to avoid adding them into upgrade ci in mechanical way. 

Here take `day2` as example, if there are some cluster configuration have both ways to run the installation in day0 or day2, in such case, generally I would suggest only cover `day0` one, unless the e2e job author ask to cover both with some reason.

While for some configurations, only can be installed via day2, e.g: `aws-ipi-outpost-day2`, we have to cover it using `day2` way.

Reply:
I understand. But there are so many profiles. It can be challenging to ensure Jian Li understand each profile and it's not that possible to train any new hire in this manner. We should establish some rules to simplify the work for maintainer and reviewers.

Reply:
Sorry, it seems right now there is no `azure-ipi-des-mini-perm-fips` in `openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml`. Let me add it. @jianlinliu

Reply:
Sorry, correct me, there is `azure-ipi-des-mini-perm-arm-f28`, so will not change anything for the comment.